tritium-metrics/src/main/java/com/palantir/tritium/metrics/TaggedMetricsExecutorService.java

```diff
@@ -29,14 +29,6 @@
 
 final class TaggedMetricsExecutorService extends AbstractExecutorService {
 
-    // 250ms minimum threshold is required to update the queued duration timer.
-    // The queued duration is an estimate based on time between a task being submitted
-    // and beginning to execute, there is always a delta between these operations, but
-    // it doesn't necessarily mean there's a queue at all. We assume anything longer than
-    // this threshold, which should be longer than pauses in most cases, is the result
-    // of queueing.
-    private static final long QUEUED_DURATION_MINIMUM_THRESHOLD_NANOS = 250_000_000L;
```

Comment:
https://github.com/palantir/tritium/pull/1230 was what originally added this threshold. Per discussions with @carterkozak & @pkoenig10 , we explicitly do not add queue metrics for cached executors in tritium clients.